### ISSUE TITLE
Aria-hidden="true" and role= 'Presentation' is not defined for decorative image present under Edit button tile.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#109]
User Experience:

If Aria-hidden attribute and role is not defined, then end user who rely on Assistive Technology will get information for the decorative images available over the screen.

Note: User credentials should NOT be included in the bug.

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the Button and select it.
8. Navigate to the "Button" dialog and select edit button.
9. Navigate to the various decorative image present over the controls.
10. Verify the issue.

 

Actual Result:

Aria-hidden="true" and role= 'Presentation' is not defined for decorative images, present under Edit button tile. Keyboard focus moves to the images and does not pass any information.

 

Expected Result:

Aria-hidden="true" and role= 'Presentation' should be defined for decorative images, present under Edit button tile.

Note:

Similar issue exists with Chrome.

 

MAS Reference:

MAS 4.1.2 - Name, Role, Value (40B)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
